README.md

```diff
@@ -344,7 +351,11 @@ You can specify the crypto-hash method in the configuration file. We use HMAC-SH
 
 A typical scenario is to replace resource ids across FHIR resources via crypto hashing. With a specific hash key, same resource ids that reside in resources and references will be hashed to a same value. There is a special case when crypto hashing a [literal reference](https://www.hl7.org/fhir/references.html#literal) element. The tool captures and transforms only the id part from a reference, for example, reference ```Patient/123``` will be hashed to ```Patient/a3c024f01cccb3b63457d848b0d2f89c1f744a3d```. In this way, you can easily resolve references across anonymized FHIR resources.
 
-# Resources
+## Encrypt method
+We use AES-CBC algorithm to transform FHIR data with an encryption key, and then replace the original value with a Base64 encoded representation of the encrypted value.
+1. The encryption key needs to be exactly 128, 192 or 256 bits long.
+2. The algorithm will generate a random and unique initialization vector (IV) for each encryption, therefore the encrypted results are different for the same input values.

```

Comment:
Will the result be different (for same input values) even with same encryptKey?

Reply:
Yes. Each encryption process will pick up a random IV and generate different result even with a same encryptKey. This is a typical senario of using Initialization Vector.